### PR TITLE
Add quote props rule, update camelcase rule

### PR DESCRIPTION
**Pull request by petesantos**
Added the quote props rule and modified the camelcase rule to prevent a rule contradiction.

Quote props can also be set to "consistent" - if one key in the object requires quotes, all keys will be quoted.

### CODE REVIEW

File: src/libs/icon-dict.js [hunk 1]
```diff
@@ -38,40 +38,40 @@ const iconDict = {
...
   'angle-double left': rotate(-90, custom(angleDoubleUp)),
   'angle-double right': rotate(90, custom(angleDoubleUp)),
-  'ban': fa(faBan),
-  'bars': custom(bars),
-  'check': fa(faCheck),
-  'clock': fa(faClock),
-  'cloud': fa(faCloud),
-  'cog': fa(faCog),
-  'copy': fa(faClone),
+  ban: fa(faBan),
```
Comment (breilly2) [inline src/libs/icon-dict.js:41]: Is this our code or something we've copied in from somewhere else? If it's ours, why would we choose `'angle down'` as a property name? If it's not ours, should we make an exception to the quote props rule?

Comment (petesantos) [reply src/libs/icon-dict.js:41]: This is our code. I am not sure what the rationale is behind having a space in the prop name. I'll change these names and chat with Isaac to make sure there isn't a reason the space is needed - thanks!

Comment (panentheos) [reply src/libs/icon-dict.js:41]: The names are a bit of a historical artifact, since we've changed icon providers a couple times.

**Review by panentheos — APPROVED** [review]
There's one more instance of an angle icon (using string interpolation) in `TopBar.js`. After that, looks good.

**Pull request merged**